### PR TITLE
add missing enum values for V4L2 camera messages

### PR DESCRIPTION
**Pull request by mikey0000**

### CODE REVIEW

File: include/system/camera.h [hunk 1]
```diff
@@ -85,6 +85,11 @@ enum {
     // request FRAME and METADATA. Or the apps can request only FRAME or only
     // METADATA.
     CAMERA_MSG_PREVIEW_METADATA = 0x0400, // dataCallback
+    CAMERA_MSG_CONTINUOUSSNAP = 0x0500,
```
Comment (ayufan) [inline include/system/camera.h]: Where we use this values? Maybe we could put them where they are used?

Comment (mikey0000) [reply include/system/camera.h]: They are used in CallbackNotifier.cpp
https://github.com/ayufan-pine64/device-softwinner/blob/nougat/common/hardware/camera/CallbackNotifier.cpp#L33

they are grouped together so it made sense to keep them all together. I might create a cameraconstants.h file then and place them in there. Thoughts?

Comment (mikey0000) [reply include/system/camera.h]: I'm going to update this to match the pine64 allwinner code for a start then I can move it to a constants file. I think this is the best move, agree?

**Comment by ayufan** [comment]
FYI https://github.com/ayufan-pine64/device-softwinner/commit/3d2ee4e8fec32fa74c7e3474724e8f23de308d90

**Comment by mikey0000** [comment]
great!

**Pull request closed without merging**